.github/contributing.rst

```diff
@@ -87,50 +87,23 @@ Anatomy of a Pull Request
 =========================
 
 There is a GitHub Pull Request template to help guide crafting a description,
-and you can liberally copy content from the commit message as needed.
+and you can liberally copy content from the commit messages as needed.
 
 If your PR isn't quite ready feel free to create it as a draft, then once
 you're all set feel free to flip it to "Ready for review".
 
+The PR can have as many commits as you want, with multiple commits being
+preferred, as it can make review easier, especially when there are many
```

Comment:
I would consider some more of the explanation from the PR description -- why not squash manually and other things mentioned there.

Reply:
No, this is how we end up with hard to follow docs. The writing style should be, keep everything very brief and to the point upfront, then elaborate later on.

Most people just want the instructions to follow and get going. We don't have to defend each point, nor do we have to convince people, right at that moment. If a would be contributor demands/feels they need that for them to follow along then they can get fucked.